source/clear-linux/get-started/bare-metal-install/cgdisk-manual-install.rst

```diff
@@ -0,0 +1,303 @@
+
+   With all the partitions now defined, you should see a list similar to what
+   is shown in figure 10:
+
+   .. figure:: figures/cgdisk-manual-install-10.png
+      :scale: 50 %
+      :alt: cgdisk - defined partitions
+
+      Figure 10: :guilabel:`cgdisk - defined partitions`
+
+#. If you are satisfied that the partition scheme is correct, you will need to
+   write this GPT to the hard drive. Select the :guilabel:`[ Write ]` button
+   or press the :kbd:`W` key and you will be prompted with:
+
+   .. code-block:: console
+
+      Are you sure you want to write the partition table to disk? (yes or no)
+
+#. Enter ``yes`` and press :kbd:`Enter` to write this data to the hard drive
+   and then select the :guilabel:`[ Quit ]` button or press :kbd:`Q` to exit
+   the :command:`cgdisk` utility and return to the |CL| manual installation
+   process.
+
+   You will see the partitions that you just created as shown in figure 11 and
+   ready for the next step in the |CL| installer setup process.
+
+#. Move your cursor to the :guilabel:`< Next >` button and press :kbd:`Enter`.
+
+   .. figure:: figures/cgdisk-manual-install-11.png
+      :scale: 50 %
+      :alt: defined partitions
+
+      Figure 11: :guilabel:`defined partitions`
+
+Set mount points
+****************
+
+The :guilabel:`Set mount points` menu will set the mount points that the |CL|
+installer will use for your |CL| installation and is shown in figure 12.
+
+.. figure:: figures/cgdisk-manual-install-12.png
+   :scale: 50 %
+   :alt: Set mount points
+
+   Figure 12: :guilabel:`Set mount points`
+
+In this menu you will need to set the mount points for the boot and root
+partitions and select to format them.
+
+#. Highlight the EFI System partition type menu entry and press the
+   :kbd:`Enter` key to edit this item. The :guilabel:`Set mount point of
+   sda1` menu will be shown and you will need to enter the following
+   information to set the mount to the :file:`/boot` directory entry and
+   enable formatting the partition by checking the :guilabel:`[ ] Format`
+   toggle field as shown in figure 13:
+
+   .. figure:: figures/cgdisk-manual-install-13.png
+      :scale: 50 %
+      :alt: Set mount point of sda1
+
+      Figure 13: :guilabel:`Set mount point of sda1`
+
+#. Do the same for the Linux filesystem partition type by highlighting the
+   :guilabel:`sda3` menu entry and entering the information shown in figure 14
+   to set the :file:`/` root directory and enable formatting:
+
+   .. figure:: figures/cgdisk-manual-install-14.png
+      :scale: 50 %
+      :alt: Set mount point of sda3
+
+      Figure 14: :guilabel:`Set mount point of sda3`
+
+   The final :guilabel:`Set mount points` menu item will look like figure 15:
+
+   .. figure:: figures/cgdisk-manual-install-15.png
+      :scale: 50 %
+      :alt: Set mount point completed
+
+      Figure 15: :guilabel:`Set mount point completed`
+
+#. Move your cursor to the :guilabel:`< Next >` button and press :kbd:`Enter`

```

Comment:
You can replace "Move your cursor to the" with "Select the" This applies to all instances.

Reply:
done